src/acceptance-test/java/org/zalando/nakadi/webservice/hila/StreamBatch.java

```diff
@@ -17,11 +18,14 @@
 
     private final Cursor cursor;
     private final List<Map> events;
+    private final JsonNode metadata;
 
     public StreamBatch(@JsonProperty("cursor") final Cursor cursor,
-                       @Nullable @JsonProperty("events") final List<Map> events) {
+                       @Nullable @JsonProperty("events") final List<Map> events,
+                       @JsonProperty("metadata") final JsonNode metadata) {
```

Comment:
I think for consistency it should have type Map. (or events should have type List<JsonNode>)